lib/database/models/typedefs/SequelizeRun.js

```diff
@@ -53,9 +53,18 @@
  * @property {string|null} pdpWorkflowParameters
  * @property {string|null} pdpBeamType
  * @property {string|null} readoutCfgUri
+ * @property {string|null} startOfDataTransfer
+ * @property {string|null} endOfDataTransfer
+ * @property {string|null} ctfFileCount
+ * @property {string|null} ctfFileSize
+ * @property {string|null} tfFileCount
+ * @property {string|null} tfFileCount
+ * @property {string|null} tfFileSize
+ * @property {string|null} otherFileCount
+ * @property {string|null} otherFileSize
+ * @property {number|null} runDuration
```

Comment:
Are you sure those are string? And not Bigint or anything?

Reply:
They get returned as number | string | null depending on if the number type can handle it or if it's empty or not. So I updated it to be according to that info